recipes/lumen/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "lumen" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5dcb6b53bf6ce3ad8d9c881dc5325917ada258efb971f257786ef1c050668424
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - lumen = lumen.command:main
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - param >=1.7.0
+    - pyct >=0.4.4
+    - setuptools >=30.3.0
+  run:
+    - python >=3.7
+    - panel >=0.11
+    - hvplot
```

Comment:
Should this also have the `pandas` and `holoviews` dependencies listed upstream?

Reply:
I have added them. 

I was not sure how much I should add under run, the bare minimum or all of the packages needed.

Reply:
Bare minimum is fine, but then I have to ask: why are they listed under `install_requires` in `setup.py`? :wink:

Reply:
That is such a good question, which I can't answer :upside_down_face: 

Thank you for your guidance and help.